packages/arb-token-bridge-ui/src/hooks/CCTP/useIsCctpAllowed.ts

```diff
@@ -0,0 +1,20 @@
+import useSWRImmutable from 'swr/immutable'
+import { useCCTP, UseCCTPParams } from './useCCTP'
+
+export function useIsCctpAllowed({ sourceChainId }: UseCCTPParams) {
```

Comment:
We already have `useCCTP` and `useAccountIsBlocked` so to better match that naming scheme:

```suggestion
export function useCCTPIsBlocked({ sourceChainId }: UseCCTPParams) {
```

The way I think of it is CCTP works by default, and this hook is here to detect the edge case (blocked)